src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/PatchCorrector.kt

```diff
@@ -0,0 +1,59 @@
+package com.bardsoftware.papeeria.backend.cosmas
+import java.util.LinkedList
+import diff_match_patch
+
+class PatchCorrector {
+    private fun getReverseOperation(operation: diff_match_patch.Operation): diff_match_patch.Operation {
+        if (operation == diff_match_patch.Operation.DELETE)
+            return diff_match_patch.Operation.INSERT
+        if (operation == diff_match_patch.Operation.INSERT)
+            return diff_match_patch.Operation.DELETE
+        return operation
+    }
+
+    fun reversePatch(patchList: LinkedList<diff_match_patch.Patch>): LinkedList<diff_match_patch.Patch> {
+        val reversePatchList = LinkedList<diff_match_patch.Patch>()
```

Comment:
use Kotlin's [mutableListOf](https://kotlinlang.org/docs/reference/object-declarations.html).
I don't think you need LinkedList for these purposes.

Reply:
listOf returns MutableList, but diff-match-patch use LinkedList